x-pack/test/security_api_integration/tests/session_idle/extension.ts

```diff
@@ -38,22 +40,26 @@ export default function ({ getService }: FtrProviderContext) {
       sessionCookie = parseCookie(response.headers['set-cookie'][0])!;
```

Comment:
__note:__ I see this test suite has been skipped in `main`, so you'll need to manually un-skip it once you merge `main` to your PR branch!